common.lic

```diff
@@ -664,16 +664,30 @@ module DRC
   def clean_instrument(settings, worn = true)
     cloth = settings.cleaning_cloth
     instrument = worn ? settings.worn_instrument : settings.instrument
-    case bput("get my #{cloth}", 'You get', 'You are already holding', 'What were you')
-    when 'What were you'
-      echo('You have no chamois cloth -- this could cause problems with playing an instrument!')
+
+    unless DRCI.get_item?(cloth)
+      message('You have no chamois cloth -- this could cause problems with playing an instrument!')
```

Comment:
```suggestion
      DRC.message('You have no chamois cloth -- this could cause problems with playing an instrument!')
```
Suggest proper DRC prefix for all the message calls.